dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ReturnTypeChangedProblem.java

```diff
@@ -16,6 +16,8 @@
 
 package com.google.cloud.tools.opensource.classpath;
 
+import com.sun.org.apache.bcel.internal.classfile.Utility;
```

Comment:
why a com.sun class?

Reply:
Fixed. It was fetching the class from rt.jar. Interesting.